src/geouned/GEOUNED/core.py

```diff
@@ -754,6 +688,77 @@ def start(self):
         logger.info(f"Translation time of solid cells {tempTime1} - {tempTime0}")
         logger.info(f"Translation time of void cells {tempTime2} - {tempTime1}")
 
+    def _decompose_geometry(self):
```

Comment:
This method must be split in 3 methods:
- 1st : from line 698 to 704  (decomposition of standard solids, and enclosures)
- 2nd: from line 702 to 742 (processing of geometry  with solids made of facets and enclosures decomposition)
- 3rd: from line 708 to 726 (conversion to solids bool definition) and 744 to 758 (conversion to enclosure de bool definition)

the 3 methods must have a flag to tell if standard of facets geometry is processed.
- if standard geometry execute:  
        * conversion to solids bool definition
        * forceNoOverlapping (lines 728 and 729) 
        * conversion to enclosure bool
      
- if facet geometry execute:
        * conversion to enclosure bool


In the start method the sequence of execution should be:
if facet geometry :
      execute 2nd method (processing of geometry  with solids made of facets and enclosures decomposition)
else   
      execute  1st method (decomposition of standard solids, and enclosures)

execute 3rd method (conversion of solids and enclosure). If flag is facet execute only conversion of enclosure


The geometry made of facets doesn't need decomposition and conversion process. The steps equivalent to these two processes are carried out in the function translate